src/args.rs

```diff
@@ -107,7 +106,9 @@ impl Args {
         };
         let date_format = String::from(matches.value_of("date-format").unwrap());
 
-        let tab_size = matches.value_of_t::<usize>("tab-size")?;
+        let tab_size = matches
+            .value_of_t::<usize>("tab-size")
+            .unwrap_or_else(|e| e.exit());
```

Comment:
This change is good, but this function no longer returns `anyhow::Error`.
So, I prefer `pub fn load() -> Args` to `pub fn load() -> Result<Args>`. Please also change it.

Reply:
Ok, I have changed. Check again please.